CHANGELOG.md

```diff
@@ -7,6 +7,11 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [python-0.4.1] - 2023-10-19
```

Comment:
You should also change the links at the bottom of this file, too; add the `python-0.4.1` tag and change the `unreleased` tag, so that it looks like

```
[unreleased]: https://github.com/promised-ai/lace/compare/python-0.4.1...HEAD
[python-0.4.1]: https://github.com/promised-ai/lace/compare/python-0.4.0...python-0.4.1
```